### PR TITLE
UI enhancements

### PR DESCRIPTION
**Pull request by rushenn**

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -106,4 +107,15 @@ gulp.task('watch', function() {
     gulp.watch(files.cssMin, ['copy-css']);
     gulp.watch(files.js, ['minify-js']);
```
Comment (kaankabalak) [inline gulpfile.js]: Looks like we don't have the `minify-js` task anymore and we merged both in the `copy-js` task. If this is the case, we might want to replace the `minify-js` with `copy-js` and remove the next line

Comment (rushenn) [reply gulpfile.js]: @kaankabalak Thank you for for pointing out the issue. I will fix it shortly.

**Comment by rushenn** [comment]
@kaankabalak Please check now.

File: lib/_sidebar-template.erb [hunk 2]
```diff
@@ -23,6 +27,15 @@
                         <%= "<%- end -\%\>" -%>
 
                             <li class='doc-link__category <%= "<%= active_keywords[0] -\%\>" -%>'>
+                                <svg class="doc-link__exapnd">
```
Comment (kaankabalak) [inline lib/_sidebar-template.erb]: Just a very minor typo on the class name here, it should be `expand` instead of `exapnd`

**Pull request merged**